seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/CheckpointEnableIT.java

```diff
@@ -225,7 +226,7 @@ public void testZetaStreamingCheckpointNoInterval(TestContainer container)
 
     @TestTemplate
     @DisabledOnContainer(
-            value = {},
+            value = {TestContainerId.FLINK_1_17, TestContainerId.FLINK_1_18},
```

Comment:
hi @PeppaPage , why disabled this? cc @TyrantLucifer

Reply:
Hi, flink 1.17 and 1.18 change the default ExecutionGraph in batch mode , it makes the dynamic-graph true, so it will skip the checkpoint because dynamic graph doesn't support.
![e80fe4b5b4d2faa81d7570e0299db9f](https://github.com/user-attachments/assets/707156e2-8f45-4aab-a000-ad5917c04002)
![a5f3d25c1c203e9b5b80ab50b1bfbf3](https://github.com/user-attachments/assets/d4d9e492-f18e-4634-85e2-a62cb69069e1)
in 1.18:
![d3057a7c564646ddbfa584c1ee79465](https://github.com/user-attachments/assets/aa649239-b386-468e-a1a0-e19499e35bfe)
in 1.15:
![6c23bafb06734c095c95776d89a64f3](https://github.com/user-attachments/assets/5f355886-38c3-4637-9e28-5758951e14af)
this problem can be fixed by setting jobmanager.scheduler: Ng
![0b3d0fbbc4859d14f28abfffb39b2be](https://github.com/user-attachments/assets/421a5a98-f72c-4db4-a070-19a326591f6b)

but i'm not sure whether it will conflict with other properties.